common/core/gpudevice.cpp

```diff
@@ -518,6 +560,50 @@ void GpuDevice::HandleHWCSettings() {
     temp_displays.swap(total_displays_);
   }
 
+  // Now set floating display configuration
+  // Get the floating display index and the respective rectangle
+  // TODO Logical display on & mosaic display on scenario
```

Comment:
I think we shouldn't support this feature for these display modes. Any reason we want to ?

Reply:
This feature is needed for logical display for some use-cases. Mosaic, may not be needed.